src/reducers/openGraph.js

```diff
@@ -12,11 +15,21 @@ export default function openGraph (state = initialState, action) {
         state,
         { isLoading: true }
       );
+    case REMOVE_OPEN_GRAPH:
+      const reducedLinks = state.links.filter(link => link.url !== action.url);
+      return Object.assign({},
+        state,
+        { links: reducedLinks }
+      );
     case RECEIVE_OPEN_GRAPH:
+      const appendedLinks = state.links.concat([{
+        url: action.openGraph.url,
+        og: action.openGraph
+      }]);
       return Object.assign({},
         state,
         {
-          [action.openGraph.url]: action.openGraph,
+          links: appendedLinks,
           isLoading: false
         }
       );
```

Comment:
In the end, we don’t use this… Could maybe be useful in the future, so I didn’t remove it.